Source/Scene/Scene.js

```diff
@@ -300,7 +302,7 @@ define([
         this._globeDepth = globeDepth;
         this._depthPlane = new DepthPlane();
         this._oit = oit;
-        this._fxaa = new FXAA();
+        this._sceneFramebuffer = new SceneFramebuffer();
```

Comment:
The FXAA file was renamed to SceneFramebuffer and the FXAA specific stuff was moved to `PostProcessLibrary`